coalib/bears/Bear.py

```diff
@@ -72,6 +81,13 @@ def supported_languages(cls):
         return (cls.LANGUAGES if isinstance(
             cls.LANGUAGES, tuple) else (cls.LANGUAGES,))
 
+    @classproperty
+    def requirements(cls):
+        """
+        :return: The requirements needed by the bear.
```

Comment:
I would mention return type here: `tuple`
Or even better: `Iterable`, as you are (presumably) able to support any iterable type, not only tuples (though they are encouraged instead of other mutable collections, frozensets are in fact also possible :D)